kea_exporter/kea.py

```diff
@@ -125,8 +131,11 @@ def setup_dhcp4_metrics(self):
             'addresses_total': Gauge(
                 f'{self.prefix_dhcp4}_addresses_total',
                 'Size of subnet address pool',
-                ['subnet', 'subnet_id']
-            )
+                ['subnet', 'subnet_id']),
+            'reservation_conflicts': Gauge(
+                f'{self.prefix_dhcp4}_reservation_conflicts',
```

Comment:
This is a unit-less accumulation and should therefore carry the `_total` suffix.

```suggestion
                f'{self.prefix_dhcp4}_reservation_conflicts_total',
```